adflow/mphys/mphys_adflow.py

```diff
@@ -1281,15 +1288,42 @@ def mphys_get_triangulated_surface(self):
 class ADflowBuilder(Builder):
     def __init__(
         self,
-        options,  # adflow options
-        mesh_options=None,  # idwarp options
-        scenario="aerodynamic",  # scenario type to configure the groups
-        mesh_type="USMesh",  # mesh type option. USMesh or  MultiUSMesh
-        restart_failed_analysis=False,  # retry after failed analysis
-        err_on_convergence_fail=False,  # raise an analysis error if the solver stalls
+        options,
+        mesh_options=None,
+        scenario="aerodynamic",
+        mesh_type="USMesh",
+        restart_failed_analysis=False,
+        err_on_convergence_fail=False,
         balance_group=None,
-        user_family_groups=None,  # Dictonary of {group: surfs} to add
+        user_family_groups=None,
+        write_solution=True,
+        res_ref=None,
     ):
+        """Create an ADflow MPhys builder
+
+        Parameters
+        ----------
+        options : dict
+            ADflow options
+        mesh_options : dict, optional
+            idwarp options, by default None
+        scenario : str, optional
+            Scenario type to configure the groups, by default "aerodynamic"
+        mesh_type : str, optional
+            mesh type option. "USMesh" or  "MultiUSMesh", by default "USMesh"
+        restart_failed_analysis : bool, optional
+            Whether to retry after failed analysis, by default False
+        err_on_convergence_fail : bool, optional
+            Whether to raise an analysis error if the solver stalls, by default False
+        balance_group : ???, optional
+            ???, by default None
```

Comment:
@lamkina, @anilyil or @ArshSaja , can one of you write a description for this balance group argument, I don't know what it does but I'm assuming it's something to do with aeropropulsive cases